fourcc.py

```diff
@@ -13,18 +13,14 @@
 mod_broadcom = re.compile(r'^#define (DRM_FORMAT_MOD_BROADCOM\w+(?=\s))', flags=re.M)
 
 with open(fourcc_include) as f:
-	for l in f.readlines():
-		ident = fmt.search(l)
-		if ident != None:
-			fmt_list.append(ident.group(1))
-			continue
-		ident = mod.search(l)
-		if ident != None:
-			mod_list.append(ident.group(1))
-			continue
-		ident = mod_broadcom.search(l)
-		if ident != None:
-			mod_list.append(ident.group(1))
+	str = f.read()
+	for ident in fmt.findall(str):
```

Comment:
Actually, we don't even need the loop anymore. Just assign straight to the list.

Reply:
I think this should work, though it puts the DRM_FORMAT_MOD_BROADCOM_SAND* formats out of order (which shouldn't be a big deal I guess, but I don't know why it happens exactly).